ambari-server/src/main/resources/common-services/ZOOKEEPER/3.4.5/package/scripts/params_linux.py

```diff
@@ -31,7 +31,7 @@
 from resource_management.libraries.functions.expect import expect
 
 # server configurations
-config = Script.get_config()
```

Comment:
We should not be modifying common-services at all. We should maintain backward compatibility here